<launch>
    <param name="robot_description" command="$(find xacro)/xacro --inorder  
        $(find mobile_robot_description)/urdf/mobile_robot.urdf.xacro"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />

    <node name="telope" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py"/>
    
    <node pkg="mobile_robot" type="cmd_vel_to_diff_drive" name="cmd_vel_to_diff_drive">
        <param name="separation_length" value="0.3"/>
    </node>

    <node pkg="mobile_robot" type="mobile_robot_odometry" name="mobile_robot_odometry"/>
    <rosparam command="load" file="$(find mobile_robot)/cfg/mobile_robot_odometry.yaml"/>
    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" 
        args="0.1 0 0 0 0 0 custom_base base_link 100" />
</launch>